include/oneapi/dnnl/dnnl_types.h

```diff
@@ -2266,6 +2266,16 @@ typedef enum {
     ///    tensor and then perform backward normalization.
     dnnl_fuse_norm_add_relu = 0x10U,
 
+    /// Use RMS normalization
+    ///
+    /// If specified:
+    ///  - on forward propagation assume mean as zero
+    ///    and use RMS norm instead of the variance
+    ///  - on backward propagation compute derivative wrt RMS norm
+    ///
+    /// When used with #dnnl_use_global_stats, only RMS norm is required to be
+    /// provided as input.
```

Comment:
Feels like header-based doc better be aligned/coincide.

Reply:
make sense, I'll take a look!

Reply:
addressed in [1b4bc9f](https://github.com/uxlfoundation/oneDNN/pull/3068/commits/1b4bc9fb3c4385541080523fbdcb567a1cf2829d) :)